src/deploy/campaign/environments.ts

```diff
@@ -172,16 +185,16 @@ export const validateEnv = (
 
   // Mainnet
   if (envLevel === "prod") {
-    requires(process.env.MOCK_MEOW_TOKEN === "false", NO_MOCK_PROD_ERR);
-    requires(process.env.STAKING_TOKEN_ADDRESS === MeowMainnet.address, STAKING_TOKEN_ERR);
     requires(!process.env.MONGO_DB_URI.includes("localhost"), MONGO_URI_ERR);
+    requires(!process.env.MOCK_Z_TOKEN, NO_MOCK_PROD_ERR);
```

Comment:
same error again

Reply:
if you do want to check it this way (with "!") you need to turn env var string into a boolean by `=== "true"` comparison first, then save it to a variable as BOOLEAN, then you can use this variable with "!" everywhere you are now